### PR TITLE
Fix the red CI

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

I've tried to update to latest eventing in order to diagnose recent CI flakyness https://testgrid.knative.dev/eventing-contrib#continuous&width=20&sort-by-flakiness=

This PR includes the fix to make the CI green again

### CODE REVIEW

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Comment by matzew** [comment]
/retest
/retest

GO IN!!!

File: kafka/channel/pkg/apis/messaging/v1alpha1/kafka_channel_conversion.go [hunk 1]
```diff
@@ -81,7 +74,13 @@ func (source *KafkaChannel) ConvertTo(ctx context.Context, obj apis.Convertible)
 					Generation:    ss.Generation,
 					SubscriberURI: subscriberURI,
 					ReplyURI:      ss.ReplyURI,
-					Delivery:      delivery,
```
Comment (aliok) [inline kafka/channel/pkg/apis/messaging/v1alpha1/kafka_channel_conversion.go:84]: Ok, we don't want the empty delivery object here. I think it makes sense.

**Review by aliok — APPROVED** [review]
/lgtm
/approve

Thanks for finding this out.

**Review by pierDipi — APPROVED** [review]
/lgtm

Thanks!

**Pull request merged**